.github/workflows/test_frameworks.yml

```diff
@@ -416,3 +416,31 @@ jobs:
           sed -i'' "s/test_langs/langs/g" tests/test_units/test_basic_app.py
       - name: Run tests
         run: nosetests
+
+  gunicorn-testsuite-20_1_0:
+    name: gunicon 20.1.0
+    runs-on: "ubuntu-latest"
+    env:
+      DD_TESTING_RAISE: true
+      # PYTHONPATH: ../ddtrace/tests/debugging/exploration/
+    defaults:
+      run:
+        working-directory: gunicorn
+    steps:
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
```

Comment:
```suggestion
          python-version: '3.10'
```

if we can

Reply:
gunicorn 20.1.0 doesn't seem to support 3.10 🙁